runtime-decompiler/src/main/java/org/jrd/backend/data/ArchiveManager.java

```diff
@@ -178,6 +187,70 @@ public File newFile(File destinationDir, ZipEntry zipEntry) throws IOException {
         return destFile;
     }
 
+    /**
+     * Packs unpacked files
+     */
+    public void pack() throws IOException {
+        // Go from end to start
+        int i = pathManager.getPathSize();
+        i -= 2;
+        for (; i >= 0; i--) {
+            // Create new zip that will contain edited files
+            String[] tmp = pathManager.get(i).split("/");
+            String path = tmpdir + "/jrd/" + tmp[tmp.length - 1] + "/";
```

Comment:
Will this work on windows? May e yes. .maybe no... I do nto know. I do not even know, if usage of File.separator would be better

Reply:
Actually the hardcoded /  are quite common. I do not insists on ussinf File.separator, but you ahve to extract many of "/" to ZIP_PATH separator. Or maybe there is something like it in zip utils?

Reply:
I'll rewrite it with File.pathSeparator. It almost definitely would not work on Windows.